basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,130 @@
 const helpers = require('./project-1');
+test('should run test', () => {
+
+})
 
 // start testing!
+it('should equal 20', () => {
+    const actual = helpers.multiplyByTen(2)
+
+    expect(actual).toEqual(20)
+})
+
+it('should equal -3', () => {
+    const expected = 2
+    const actual = helpers.subtractFive(expected)
+
+    expect(actual).toEqual(-3)
+})
+
+it('should equal same length', () => {
+    const actual = helpers.areSameLength('rainbow', 'seventy')
+
+    expect(actual).toEqual(true)
+})
+
+it('should equal true', () => {
+    const actual = helpers.areEqual(2,2)
+    expect(actual).toEqual(true)
+})
```

Comment:
Have a test to also make sure the function returns false when the inputs are not equal.